### ISSUE TITLE
do not name repositories after specific classes

### ISSUE THREAD

**Issue by ghost** [archclassroom/archclassroom.github.io#11]
Class materials are meant to be resources used in many classes of the same topics. Repositories should be named after the topic rather than a specific class.

Rename the repository git-for-gits to git-vcs.

Rename the repository python-beginners to python. While a specific class may be for beginners, we should not restrict the materials to beginner level stuff only. Allow for possible classes on intermediate and even advanced levels. If this repository is renamed, inform pulec so they can update their remotes.

This issue is related to #1 I think.

**Comment by polyzen** [comment]
Blocked by #1.

**Comment by meskarune** [comment]
I'm going to close since it seems like a duplicate of #1
